src/bookshelf/src/bksw_wm_base.erl

```diff
@@ -15,28 +15,124 @@
 %% under the License.
 %%
 %% @author Ho-Sheng Hsiao <hosh@chef.io>
-%% @author Tim Dysinger <dysinger@chef.io>
+%% @author Tim Dysinger   <dysinger@chef.io>
+%% @author Lincoln Baker  <lbaker@chef.io>
 
 -module(bksw_wm_base).
 
 %% Complete webmachine callbacks
--export([init/1,
+-export([finish_request/2,
+         init/1,
          is_authorized/2,
-         finish_request/2,
+         malformed_request/2,
          service_available/2]).
 
 %% Helper functions
 -export([create_500_response/2]).
 
+-ifdef(TEST).
+-compile([export_all, nowarn_export_all]).
+-endif.
+
 -include("internal.hrl").
 
 %%
 %% Complete webmachine callbacks
 %%
 
+% why is this here?
+-include_lib("eunit/include/eunit.hrl").
+
+% number of seconds in 1 week
+-define(WEEK, 604800).
```

Comment:
s/WEEK/SECONDS_IN_WEEK. Added bonus of making the comment unnecessary.

Reply:
Done.